multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -48,7 +48,7 @@
 @SuppressWarnings({"unchecked", "rawtypes"}) // core’s fault
 public class WorkflowMultiBranchProject extends MultiBranchProject<WorkflowJob,WorkflowRun> {
 
-    static final String SCRIPT = "jenkins.groovy";
+    static final String SCRIPT = "Jenkinsfile";
```

Comment:
Not sure it is as important as for Literate. The content of this file is really specific to Jenkins-workflow whereas for Literate, the marker file content is descriptive and could be used in another context.